### DISCUSSION TITLE
Make use of specify_rows endpoint

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5788] [Feature Requests]
It is going to be more performant and efficient for many use cases, although also more complicated to deal with thus pros/cons have to be weighted

In my testing, fetching list of queries for the query dialog using `/api/specify_rows` was 10x faster than using `/api/specify`

**Comment by grantfitzsimmons** [comment]
I just feel like @realVinayak would be interested in this
